BridgeCareApp/BridgeCare/Models/InventoryItemModel.cs

```diff
@@ -0,0 +1,18 @@
+﻿namespace BridgeCare.Models
+{
+    public class InventoryItemModel
+    {
+        public InventoryItemModel(string columnKey, string AlphaNumericId, string IdName)
+        {
+            ColumnName = columnKey;
+            Id = AlphaNumericId;
+            ViewName = IdName;
+        }
+
+        public string ColumnName { get; set; }
+        public string Id { get; set; }
+        public string ViewName { get; set; }
+
+        public dynamic Value { get; set; }
```

Comment:
Use of dynamic types is discouraged. If this can be anything but dynamic, then it should be changed.

Reply:
ok, as it is all the values except the key are interpreted as strings so it does not gain much using a runtime polymorphic

Reply:
changed to string